docs/OV_Runtime_UG/ov_dynamic_shapes.md

```diff
@@ -0,0 +1,110 @@
+#Dynamic Shapes
```

Comment:
You didn't include this document to OpenVINO documentation. Please include it into OpenVINO UG.